sdf/final-project/overview.md

```diff
@@ -1,25 +1,16 @@
 # SDF Final Project Rubric
-
 This is intended to be filled out by staff members of DPI Tech Apprenticeship program.
 
-Date/Time:
-
-Trainee Name:
-
-Project Name:
-
-Reviewer Name:
+- Date/Time:
+- Trainee Name:
+- Project Name:
+- Reviewer Name:
 
 ## Overall Score
-
-[Technical](./technical.md) (/30):
-
-[Business](./business.md) (/10):
-
-[Presentation](./presentation.md) (/20):
-
+- [Presentation](./presentation.md) (/15):
+- [Product](./product.md) (/15):
+- [Technical](./technical.md) (/30):
 ---
```

Comment:
Should we also include a total for Bonus points here? I think you can get up to 15 bonus points across all categories, if I counted correctly.

Reply:
bonuses can just get added for each category